liquibase/liquibase.github.com
# Adding Installer Blog

## What I Did
Added a blog about our new Installers with an image and an embedded YouTube video demo.
## How To Test It
Blog should appear on the Blog list. 
Blog should have a working embedded YouTube video.
Blog should have an image showing the files involved in the install.

* PLEASE push today. I know that leadership will want to see this live before the Friday morning standup. *
